### DISCUSSION TITLE
Can't get entries

### DISCUSSION THREAD

**Discussion by evolvewd** [harness-software/wp-graphql-gravity-forms#254] [Q&A] [answered]
Hello, querying entries i've nothing back
![image](https://user-images.githubusercontent.com/3524394/167025256-0466a426-db0b-4f76-b963-5183bfe9d753.png)
but i've several submitted entries.

Doing a similar query on forms i get back correctly the active forms.
![image](https://user-images.githubusercontent.com/3524394/167025407-141dd2f5-0f14-431f-9b9c-cb3b0b0c0114.png)

WP 5.9.3
GF 2.6.1.1
WP GraphQL 1.8.1
WP GraphQL for GF 0.11.0

**Comment by justlevine** [comment, marked as answer]
Hey @evolvewd - are you authenticated? (Note the small green dot on the user icon)
![image](https://user-images.githubusercontent.com/29322304/167073677-77bb5b6d-ba89-4121-aa91-0a35a0dc7aaf.png)

Entries can only be viewed by an admin /the user who submitted them.

**Reply by evolvewd** [reply]
Thanks a lot. My bad, now works perfectly

**Reply by DanePete** [reply]
This is a silly ui, just silly :)

**Reply by justlevine** [reply]
@DanePete , that's from [WPGraphQL core](https://github.com/wp-graphql/wp-graphql). If you have a suggestion to improve it, I'm sure they'd be receptive to it.
